docs/docs/community/index.mdx

```diff
@@ -111,6 +111,10 @@ Our contributing and community guidelines take inspiration from the following pr
 - [Julia Language](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md)
 - [nteract](https://github.com/nteract/nteract/blob/main/CONTRIBUTING.md)
 
+Our gitflow process takes inspiration from the following projects:
+
+- [Bokeh](https://github.com/bokeh/bokeh/wiki/BEP-6:-Branching-Strategy)
+
```

Comment:
My previous comment might have been confusing - The format here is correct, but this should be in the `Release and branching strategy` doc not in the community index 😓  apologies for the confusion